chanzuckerberg/idseq-web
# [Maps] Add "click out to de-preview", stretch map side margins, tweak bubble sizes, add No Samples banner

### Description
- Add feature to "de-preview" when you click a blank part of the map.
- Move around the center pane so that you don't get NarrowContainer on the map (so that it fills in the white space to the left and right of the map).
- Tweak bubble sizing to make some bubbles larger (derived from eye-balling it and Design/UX feedback).
- Add "No samples/projects found" message to the top of the map with a clear all button.
- Add currentDisplay to the saved URL state.
- Exit preview mode when you filter out a location.
- Vertically align the zoom controls and info button.

### Demo
![Jun-17-2019 11-19-19](https://user-images.githubusercontent.com/5652739/59627275-ae2a3d80-90f2-11e9-8fa2-9daebc809101.gif)
![Jun-17-2019 18-24-33](https://user-images.githubusercontent.com/5652739/59646764-3299b200-912d-11e9-984a-9db656bb1d9c.gif)
![Jun-17-2019 18-23-54](https://user-images.githubusercontent.com/5652739/59646757-2a417700-912d-11e9-97fe-486a84e566b8.gif)

### Tests
- Go to data discovery, stretch out your window size to see that the map stretches out as well.
- Click a bubble to preview and see the right sidebar update. Then click a blank area of the map to see it exit preview mode.
